app/app.pro

```diff
@@ -84,9 +84,10 @@ icon256.files = icons/256x256/harbour-books.png
 icon256.path = $$TARGET_ICON_ROOT/256x256/apps
 INSTALLS += icon256
 
-CONFIG += sailfishapp_i18n sailfishapp_i18n_idbased
+CONFIG += sailfishapp sailfishapp_i18n sailfishapp_i18n_idbased
```

Comment:
I don't think this change is necessary, `sailfishapp` is already added to `CONFIG` at line 2.